examples/image-to-text/run_pipeline.py

```diff
@@ -169,18 +207,14 @@ def main():
         generator.model = wrap_in_hpu_graph(generator.model)
 
     if args.quant_config:
-        import habana_quantization_toolkit
-
-        habana_quantization_toolkit.prep_model(generator.model)
-
-        htcore.hpu_initialize(generator.model)
```

Comment:
@tthakkal you removed this initialization. Don't we need it anymore?

Reply:
Added back the code, interestingly its missing in text-generation sample too. but it works without.